components/cdc/src/endpoint.rs

```diff
@@ -347,10 +374,14 @@ pub struct Endpoint<T, E> {
     region_read_progress: RegionReadProgressRegistry,
 
     // Metrics and logging.
-    min_resolved_ts: TimeStamp,
-    min_ts_region_id: u64,
-    resolved_region_count: usize,
-    unresolved_region_count: usize,
+    txnkv_min_resolved_ts: TimeStamp,
```

Comment:
Maybe we can use the "keyspace" to handle metrics for different mode. Every metric would be hold in a map, key by the "keyspace".